tools/ci_build/github/azure-pipelines/mac-react-native-ci-pipeline.yml

```diff
@@ -0,0 +1,297 @@
+parameters:
+- name: NpmPublish
+  displayName: 'NPM packages publish configuration'
+  type: string
+  values:
+  - 'nightly (@dev)'
+  - 'release candidate (@rc)'
+  - 'production (@latest)'
+  - 'custom'
+  default: 'nightly (@dev)'
+
+variables:
+  build_config: Release
+  ${{ if eq(parameters.NpmPublish, 'nightly (@dev)') }}:
+    # 'dev' will skip generating onnxruntime-common package when it's not updated
+    # since react-native e2e always requires both onnxruntime-common and onnxruntime-react-native packages,
+    # use 'custom' type mode here.
+    npm_packaging_mode: '-dev.$(Get-Date -Format yyyyMMdd)-$(git rev-parse --short HEAD)'
+  ${{ if eq(parameters.NpmPublish, 'release candidate (@rc)') }}:
+    npm_packaging_mode: 'rc'
+  ${{ if eq(parameters.NpmPublish, 'production (@latest)') }}:
+    npm_packaging_mode: 'release'
+  ${{ if eq(parameters.NpmPublish, 'custom') }}:
+    npm_packaging_mode: '$(VersionSuffix)'
+
+jobs:
+- template: templates/android-java-api-aar.yml
+  parameters:
+    buildConfig: '${{variables.build_config}}'
+    buildSettings: '$(Build.SourcesDirectory)/tools/ci_build/github/js/react_native_e2e_mobile_aar_build_settings.json'
+    includedOpsConfig: '$(Build.SourcesDirectory)/tools/ci_build/github/android/mobile_package.required_operators.config'
+    artifactName: 'onnxruntime-android-mobile-aar'
+    job_name_suffix: 'For_React_Native'
+    pool_name: 'Linux-CPU-2019'
+
+- job: ReactNative_CI
+  pool:
+    vmImage: 'macOS-11'
+  dependsOn:
+  - Android_Java_API_AAR_Packaging_For_React_Native
+  timeoutInMinutes: 120
+  steps:
+  # Onnx has no 3.9 python package available yet, need to use python 3.8 to avoid build onnx package
+  # pythonVersion can be updated in Azure pipeline settings
+  # https://dev.azure.com/onnxruntime/onnxruntime/_build?definitionId=53
+  - task: UsePythonVersion@0
```

Comment:
This is the link to Android CI, change to
[https://dev.azure.com/onnxruntime/onnxruntime/_build?definitionId=188](https://dev.azure.com/onnxruntime/onnxruntime/_build?definitionId=188) ?